index.js

```diff
@@ -20,9 +21,9 @@ function inCommand (cmd) {
 }
 
 exports.inPublish = function () {
-  return inCommand(/^pu(b(l(i(sh?)?)?)?)?$/)
+  return inCommand(/^(?:pre)?pu(b(l(i(sh?)?)?)?)?$/)
 }
 
 exports.inInstall = function () {
-  return inCommand(/^i(n(s(t(a(ll?)?)?)?)?)?$/)
+  return inCommand(/^(?:pre)?i(n(s(t(a(ll?)?)?)?)?)?$/)
```

Comment:
IIRC that was just for consistency with the other change. I don't believe `lerna` demands it, and have no idea whether `yarn` might.